telecine/build.gradle

```diff
@@ -27,7 +27,7 @@ android {
     applicationId 'com.jakewharton.telecine'
 
     minSdkVersion 21
-    targetSdkVersion 22
+    targetSdkVersion 25
```

Comment:
This has to remain at 22 to prevent the need to use runtime permissions and a bug on Android 6.0 which didn't properly grant access to apps to draw on top of system windows.